setup.py

```diff
@@ -98,7 +98,7 @@ def get_extras_require() -> Dict[str, List[str]]:
             "thop",
             "skorch",
             "stable-baselines3>=0.7.0",
-            "catalyst<21.3",
+            "catalyst",
```

Comment:
Could you clarify the minimum version of Catalysts? The integration raises `ImportError` if the Catalyst version is less than 21.3.

```suggestion
            "catalyst>=21.3",
```